expr/service_test.go

```diff
@@ -43,6 +43,42 @@ func TestServiceExprMethod(t *testing.T) {
 	}
 }
 
+func TestServiceExprError(t *testing.T) {
+	var (
+		errorFoo = &ErrorExpr{
+			Name: "foo",
+		}
+		errorBar = &ErrorExpr{
+			Name: "bar",
+		}
+	)
+	cases := map[string]struct {
+		name     string
+		expected *ErrorExpr
+	}{
+		"exist": {
+			name:     "foo",
+			expected: errorFoo,
+		},
+		"not exist": {
+			name:     "baz",
+			expected: nil,
+		},
```

Comment:
Would it be possible to add a test case where the `Root` expression has an error with the given `name` and `service.Error()` returns the correct error expression? Right now, your test cases does not cover this condition - https://github.com/goadesign/goa/blob/v2/expr/service.go#L70

Reply:
Woops, https://github.com/goadesign/goa/pull/2024/commits/7d3a3363dda4cd999c1c795bc9e2d556e4d7368c adds the test.